src/service-worker/src/armada/app-version.ts

```diff
@@ -6,9 +6,11 @@ import {DebugHandler} from '../debug';
 import {IdleScheduler} from '../idle';
 import {Manifest} from '../manifest';
 import {MsgAny} from '../msg';
+import {sha1Binary} from '../sha1';
```

Comment:
do we use sha1?

Reply:
We seem to use the same thing in, `src/service-worker/src/armada/assets.ts` so I just used the same here.

Reply:
Fixing this in #21